### PR TITLE
Update readme (again)

### LINKED ISSUE #1944: Update README again
Description:

The output from `yarn build:local --overwrite=true` differs from the one from `yarn start:new` which is not documented on the readme. 

```
yarn build:local --overwrite=true
yarn run v1.13.0
$ node tasks/build/local/build --overwrite=true
Created Account: { name: 'username',
  type: 'local',
  address: 'cosmos1dgk0y0q3swhsqcjgvqmznw0gnln06mhfv5kxar',
  pub_key:
   'cosmospub1addwnpepqvmccel0z9uej7zv94v0r5h8zmyywl8wug8tfj3tx3zgcwzvg6ymsdutcnk',
  mnemonic: '...'}

    🎉  SUCCESS 🎉

To start Voyager with 1 local node please run:
  yarn start local-testnet

Default account:
  username: 'username'
  password: '1234567890'

✨  Done in 1.34s.
```
```
yarn start:new
Created Account: { name: 'account-with-funds',
  type: 'local',
  address: 'cosmos1spa47t6qqcg3yu73pyvfngwzw2hc6krl2efn3a',
  pub_key:
   'cosmospub1addwnpepqd37w8afe5geklcr37suq3jufzxgu8d2vt9tgm55mkphxagfvxzqyh4vrge',
  mnemonic:
   '...' }
```
Side note, do we still need to build:local in order to make voyager work ?

### PR DESCRIPTION
**Pull request by fedekunze**
Closes #1944

**Description:**

Thank you! 🚀

---

For contributor:

- [x] Added entries in `CHANGELOG.md` with issue # and GitHub username
- [x] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -129,87 +121,60 @@ yarn backend:fixed-https
...
-cp ~/.gaiad-testnet/config/{genesis.json,config.toml} builds/testnets/local-testnet/
+yarn build:ui
 ```
 
-Enter your local node as a seed:
+If you want to set a particular Stargate or RPC endpoints
 
 ```bash
-sed -i.bak 's/seeds = ""/seeds = "localhost"/g' ./builds/testnets/local-testnet/config.toml
+STARGATE=<https://StargateAddress:port> RPC=<https://RPRAddress:port> yarn build:ui
```
Comment (faboweb) [inline README.md]:
```suggestion
STARGATE=<https://StargateAddress:port> RPC=<https://RPCAddress:port> yarn build:ui
```

Comment (faboweb) [reply README.md]: I am not sure this works.

File: README.md [hunk 2]
```diff
@@ -129,87 +121,60 @@ yarn backend:fixed-https
...
+  }
 ```
 
-If you want to set a particular Stargate or RPC endpoints
+To import the account to Voyager, go to the `Import with seed` option on the `Sign in`, and paste the `mnemonic` value from above on the `Seed Phrase` field.
 
-```bash
-STARGATE=<https://StargateAddress:port> RPC=<https://RPRAddress:port> yarn build:ui
-```
+**Note:** Running `yarn start:new` overwrites all previously generated local accounts, meaning that you will have to import the account every time ! Use `yarn start` if you want to keep your account.
```
Comment (faboweb) [inline README.md]: 👍

File: README.md [hunk 3]
```diff
@@ -129,87 +121,60 @@ yarn backend:fixed-https
...
-
-### Running several nodes
+#### Running several nodes
 
-This command will build and run several nodes at once on the local testnet. All nodes will be validators:
+This command will build and run several nodes at once on the `local-testnet`. All nodes will be validators:
 
 ```bash
-yarn start local-testnet <number>
+yarn start <number>
```
Comment (faboweb) [inline README.md]: did you test this?

File: README.md [hunk 4]
```diff
@@ -269,7 +234,7 @@ node_modules/.bin/tape test/e2e/init.js
 To produce an up-to date documentation you can run:
 
 ```bash
-$ yarn doc
+yarn doc
```
Comment (faboweb) [inline README.md]: we will remove this #2023

**Comment by fedekunze** [comment]
updated, multiple validators command fails

File: README.md (hunk 1 above)
Comment (fedekunze) [reply README.md]: cc: @sabau  ?

File: README.md (hunk 3 above)
Comment (fedekunze) [reply README.md]: fails

File: README.md [hunk 5]
```diff
@@ -257,25 +204,13 @@ If you would like to run a single test please set the TEST variable (Unix system
...
 yarn pack
 node_modules/.bin/tape test/e2e/init.js
 ```
 
-### Documentation
-
-To produce an up-to date documentation you can run:
-
-```bash
-$ yarn doc
```
Comment (sabau) [inline README.md]: saad

File: README.md [hunk 6]
```diff
@@ -129,95 +119,52 @@ yarn backend:fixed-https
...
 
-Write down the 24 word secret phrase to be able to import an account that holds tokens later on.
-
-Copy the configuration files (assuming you are in the Voyager dir):
+If you want to set a particular `Stargate` (Cosmos light client) or Tendermint's `RPC` endpoints:
 
 ```bash
-mkdir builds/testnets/local-testnet
-cp ~/.gaiad-testnet/config/{genesis.json,config.toml} builds/testnets/local-testnet/
+STARGATE=<https://StargateAddress:port> RPC=<https://RPCAddress:port> yarn build:ui
```
Comment (sabau) [inline README.md]: fixing it

**Pull request merged**